rebar.config

```diff
@@ -5,7 +5,9 @@
         {rocksdb, "0.12.0"},
         {cowboy, "1.0.4", {git, "https://github.com/ninenines/cowboy.git", {tag, "1.0.4"}}},
         {jsx, {git, "https://github.com/talentdeficit/jsx.git", {tag, "2.8.0"}}},
-        {jesse, {git, "https://github.com/for-GET/jesse.git", {tag, "1.5.0-rc2"}}}
+        {jesse, {git, "https://github.com/for-GET/jesse.git", {tag, "1.5.0-rc2"}}},
+        {gproc, "0.6.1"},
+        {jobs, "0.7.0"}
```

Comment:
I am not sure why jobs is not in the rebar.lock.

Reply:
Not sure either